packages/@aws-cdk/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -85,9 +86,16 @@ export class Accelerator extends cdk.Resource implements IAccelerator {
   constructor(scope: Construct, id: string, props: AcceleratorProps = {}) {
     super(scope, id);
 
+    const maxResourceNameLength = 64;
+    const name = cdk.FeatureFlags.of(this).isEnabled(cxapi.GLOBAL_ACCELERATOR_SAFE_RESOURCE_NAME) ?
```

Comment:
Because this isn't used on a stateful resource, I'm not actually sure we need the feature flag here. Checking in with other team members on this.

Reply:
Ok, let me know. I've added the feature flag because the issue is similar to [this one](https://github.com/aws/aws-cdk/issues/24654), where a feature flag was [requested](https://github.com/aws/aws-cdk/issues/24654#issuecomment-1473826663).

Reply:
I've removed the FF.
Once this has been reviewed and accepted I will align the [other PR](https://github.com/aws/aws-cdk/pull/24786) as well.